app/templates/components/dialogs/prioritize-build-modal.hbs

```diff
@@ -0,0 +1,66 @@
+<Modal
+  class="repo-actions-modal"
+  @isVisible={{@isOpen}}
+  @onClose={{fn @onClose}}
+>
+  <div class="repo-actions-modal__top">
+    <h3 class="repo-actions-modal__header">
+      Prioritize your build
+    </h3>
+    <button
+      type="button"
+      title="Close modal"
+      class="pointer repo-actions-modal__close-button"
+      autofocus={{this.focusOnModal}}
+      {{on 'click' (fn @onClose)}}
+    >
+      <SvgImage @name="icon-failed" @class="icon" />
+    </button>
+  </div>
+  <div class="repo-actions-modal__body">
+    <p>
+      Bypass your scheduled build by fast tracking this one.
+    </p>
+    <p class="label-align">
+      SELECT AN OPTION
+    </p>
+    <div>
+      {{#each this.options as |option|}}
+        {{#let (eq this.selectionKey option.key) as |selected|}}
+          <button
+            class="repo-actions-item {{if selected 'repo-actions-item--selected'}}"
+            onclick={{action (mut this.selectionKey) option.key}}
+          >
+            <span class="repo-actions-item__radio"></span>
+            <span class="repo-actions-item__description">
+              {{option.description}}
+            </span>
+          </button>
+        {{/let}}
+      {{/each}}
+    </div>
+  </div>
+  <br />
+  <div class="repo-actions-modal__controls">
+    <button
+      type="button"
+      class="button button--blue"
+      onclick={{action (perform this.increasePriorityTask)}}
+    >
+    {{#if this.isLoading}}
+      <LoadingIndicator class="loading-indicator"/><span>Prioritizing the build</span>
+    {{else}}
+    <span>Prioritize build</span>
+    {{/if}}
+    </button>
+    {{#if (not this.isLoading)}}
+    <button
+      role="button"
+      class="button button--red"
+      {{on 'click' (fn @onClose)}}
+    >
+      Cancel
+    </button>
+    {{/if}}
```

Comment:
Indentation levels seems to be a bit messy here. Could you please indent the blocks so that nested ones were deeper and visualize hierarchy?

Reply:
Done.